### ISSUE TITLE
Unable to open 'lib object_patch.dart': cannot read property of undefined and TypeErrors

### ISSUE THREAD

**Issue by Shaydoth** [Dart-Code/Dart-Code#1386]
In a program that was recently working perfectly fine, upon running the app as normal the launch screen would show, then it would error and give me this into the console:

<img width="555" alt="screenshot 2019-01-10 at 07 59 58" src="https://user-images.githubusercontent.com/27705628/50954260-bcbe5a00-14ad-11e9-8a88-5db7f08e3390.png">

And yet the last time I ran it all that was different was that there weren't images within the runner of the application, to give the launch screen an image. Any ideas why this is happening?

I just installed dart onto this machine and am using visual studio code with all the most up to date version installed.

**Comment by DanTup** [comment]
Are you using VS Code insiders (1.31)? This looks like a VS Code bug at a glance, and their issue tracker seems to have a lot of "cannot read property 'x' of undefined" errors in it:

![screenshot 2019-01-10 at 8 38 53 am](https://user-images.githubusercontent.com/1078012/50956302-3e64b680-14b3-11e9-95ed-94209ce68ac7.png)

If it is Insiders, I think it is worth raising there (and confirming it doesn't happen in stable).

**Comment by DanTup** [comment]
I get a different issue in insiders :(

> Cannot read property 'adapter' of undefined

I'll open an issue about that one!

**Comment by Shaydoth** [comment]
I am currently using v1.24.1, not insiders. After more tests it seems to have nothing to do with adding a launch screen image, as removing it didn't remove the error. It probably is a vs code bug. I could try running it in android studio to see if it makes any difference, to narrow down the error.

**Comment by DanTup** [comment]
Ah, ok! The error is occurring inside VS Code code, so it definitely won't repro in Android Studio.

I would definitely open an issue in the VS Code repo with the screenshot (or maybe test it in Insiders first, it's possible it's gone). Even if we have a bug that's causing it, I don't think VS Code should fail like that. They may be able to provide tips to debug.

(If you do, please paste the link back here so I can help investigate if required)

**Comment by Shaydoth** [comment]
Yep, it seems to have been fixed in VS insiders, so no problems!
